js/main.js

```diff
@@ -62,4 +114,50 @@
 		map.invalidateSize();
 	}
 
+	function hex2rgb(hex){
+		var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
```

Comment:
There's a version of this that uses slightly less regex: http://stackoverflow.com/a/9304367/